### PR TITLE
hook in update redis cache

### PR DESCRIPTION
**Pull request by matt-gregory**

### CODE REVIEW

File: service/organisation-service.js [hunk 1]
```diff
@@ -46,6 +50,20 @@ async function deleteOrganisation (orgId) {
         })
 }
 
+async function rebuildRedisOrganisationsCache () {
+    debugger
```
Comment (bb128b) [inline service/organisation-service.js]: What is debugger?

Comment (richmilton) [reply service/organisation-service.js]: It is meant to pause js apps (like a breakpoint) probs shouldn't be here

Comment (matt-gregory) [reply service/organisation-service.js]: debugger is a bug!

**Comment by VassilAtanasov** [comment]
We could move CacheRebuildTestController.cs  in Vmd.TestSupportApi so TDG is less dependant on other services

File: service/organisation-service.js [hunk 2]
```diff
@@ -10,6 +11,7 @@ async function createOrganisation (orgType, payload) {
 
     return axios.post(url, payload)
         .then((response) => {
+            rebuildRedisOrganisationsCache()
```
[the remark this answers is not in the document]
Comment (matt-gregory) [reply service/organisation-service.js]: :+1:

**Comment by matt-gregory** [comment]
> We could move CacheRebuildTestController.cs in Vmd.TestSupportApi so TDG is less dependant on other services

Not really. I considered this, but the cache population is driven from Vmd.Admin so trying to do this in test support api could lead to race conditions pushing stuff into the cache

**Pull request merged**